### ISSUE TITLE
_puppet$ is not defined

### ISSUE THREAD

**Issue by dillionverma** [aidenybai/million#561]
### What version of `million` are you using?

2.6.0-beta.10

### Are you using an SSR adapter? If so, which one?

Nextjs

### What package manager are you using?

yarn

### What operating system are you using?

Mac

### What browser are you using?

Arc

### Describe the Bug

I run `yarn dev`

I see error

```bash
- error components/user-auth-form.tsx (141:16) @ _puppet$
- error ReferenceError: _puppet$ is not defined
    at _puppet$2.svg (./components/user-auth-form.tsx:286:89)
  139 |   _3
  140 | }) => {
> 141 |   return <slot><_puppet$ _={_} _2={_2} />{_3}</slot>;
      |                ^
  142 | }, {
  143 |   svg: false,
  144 |   shouldUpdate: (a, b) => a?._ !== b?._ || a?._2 !== b?._2 || a?._3 !== b?._3
```

I narrowed it down by commenting out code in my component. I commented this out and it worked.

https://github.com/dillionverma/llm.report/blob/5e06c37d0a09fcf745a809e0d09bc4b7f8526111/components/user-auth-form.tsx#L122-L131

```jsx
    {/* <div className="relative">
        <div className="absolute inset-0 flex items-center">
          <span className="w-full border-t" />
        </div>
        <div className="relative flex justify-center text-xs uppercase">
          <span className="bg-background px-2 text-muted-foreground">
            Or continue with
          </span>
        </div>
      </div> */}
```

And then it works fine

```bash
- event compiled client and server successfully in 1242 ms (1728 modules)
- wait compiling /api/auth/[...nextauth] (client and server)...
- event compiled successfully in 51 ms (875 modules)
```

### What's the expected result?

`yarn dev` should start server

### Link to Minimal Reproducible Example

```bash
git clone https://github.com/dillionverma/llm.report
cd llm.report
yarn
yarn dx

# uncomment million.js in next.config.mjs

yarn dev 
```

### Participation

- [ ] I am willing to submit a pull request for this issue.

**Comment by CamiloGPT** [comment]
I have the same issue.

```javascript

const _puppet$2 = /*million:transform*/_block$(({
  _,
  _2
}) => {
  return <slot><_puppet$ _={_} />{_2}</slot>; // <------- this error 
}, {
  svg: false,
  shouldUpdate: (a, b) => a?._ !== b?._ || a?._2 !== b?._2
});

```
I'm using SVGR Playground to create SVG component:
https://react-svgr.com/playground/?typescript=true

I also had warnings about {...props} in svg.

```javascript
import { SVGProps } from "react";

function SvgComponent(props: SVGProps<SVGSVGElement>) {
  return (
    <svg
      {...props}   //  <----- this, compile error message stating that it is not fully supported.
      viewBox="0 0 91 63"    
      >
      <path d="..." />
    </svg>
  );
}
```

**Comment by aidenybai** [comment]
@dillionverma @CamiloGPT Pushed a fix in `v2.6.0-beta.11`, let me know if it fixes the issue

**Comment by dillionverma** [comment]
tried it again - fix worked 🎉

million is back in llm.report
